docs/src/pages/components/hero.mdx

```diff
@@ -1,4 +1,199 @@
+import { Link } from 'gatsby';
+
+<br />
+
 # Hero
 
+The `Hero` component is used to display a website hero which spans from edge to edge of available
+space while centering it's content. Graphics, buttons and any other element can be used inside of a
+`Hero`.
+
+Aside from the `Hero` component itself, it has two additional components that can be deconstructed
+from it.
+
+```jsx
+    import { Hero } from '@retailmenot/anchor';
+
+    const { Title, Subtitle } = Hero;
+```
+
+Depending on how your environment is set up, you could directly access these sub-components directly
+in your jsx as shown below. This example uses the `colors` export from **Anchor** which is the basis for <Link to="/theme">themes</Link>,
+as well as the <Link to="/components/button/">`Button`</Link> component.
+
+```tsx live
+    <Hero background={colors.grapePurchase.base}>
+        <Typography tag="div" p="4">
```

Comment:
There should be a way of adding padding to the Hero itself but unfortunately I overlooked that. So instead of adding a Typography with padding, I'd set the `minHeight` of the `Hero` to, say, `12.5rem`, and then in the future we can change it.

Reply:
Done.